packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerCalendarCellStyles.ts

```diff
@@ -1,63 +1,42 @@
 import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
 import { DatepickerVariables } from './datepickerVariables';
 import { DatepickerCalendarCellStylesProps } from '../../../../components/Datepicker/DatepickerCalendarCell';
-import { getBorderFocusStyles } from '../../getBorderFocusStyles';
+
+const cellStyles = (p: DatepickerCalendarCellStylesProps, v: DatepickerVariables) => ({
```

Comment:
What is a purpose of having `cellStyles` function? I see only a single usage of it